docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.8"
+services:
+  saturn-node:
+    image: ghcr.io/filecoin-saturn/l1-node:${SATURN_NETWORK:-test}
+    container_name: saturn-node
+    restart: always
```

Comment:
@DiegoRBaquero do we want to default to `unless-stopped` instead?

Reply:
Yes